e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/multiple-contract-policy/multiple-contract-policy.spec.js

```diff
@@ -329,6 +354,8 @@ context('Insurance - Policy and exports - Multiple contract policy page - As an
         });
 
         multipleContractPolicyPage[TOTAL_SALES_TO_BUYER].input().should('have.value', '1000');
+
+        multipleContractPolicyPage[MAXIMUM_BUYER_WILL_OWE].input().should('have.value', '500');
```

Comment:
not sure if this is tech debt, but can we get these values from a constant?

Reply:
Great idea @Zainzzkk - i've created an application fixture file to be used moving fowards (updated notes in the PR description). Updated single and multiple policy form submission functions, and the tests we have for "values are populated when revisiting the page"